### DISCUSSION TITLE
Does Mozc support wasm build

### DISCUSSION THREAD

**Discussion by qingqingkzhou** [google/mozc#570] [General]
I have seen `wasm` being defined in several BUILD. but can't find it in [BUILD.bazel](https://github.com/google/mozc/blob/master/src/BUILD.bazel#L128). Or is there steps on how to build for wasm? Thank you.

**Comment by hiroyuki-komatsu** [comment]
Hi,

Yes, the code contains some settings for `wasm`, but it's not completed.
So there is no supported way for wasm build.

Thank you,

**Comment by andrawsf** [comment]
Is this now supported?

**Comment by tcjiang** [comment]
Hi there, I would like to contribute to the wasm build part.
I am trying to use emscripten to build the wasm to make Mozc running on ChromeOS, as previous Nacl. 
Could you please confirm is this doable at this stage? I see the macos, linux and windows build uses gui package based qt, which does not support Android nor wasm. I am not sure what to build instead of gui?  Thanks!

**Comment by hiroyuki-komatsu** [comment]
Hi,

We would support wasm as a development library like Android, but we don't have a plan to support wasm as a full application with GUIs like Linux/Win/Mac. I'm personally not sure if it's feasible or not. Thanks!
